packages/components/src/utils/index.ts

```diff
@@ -90,12 +90,25 @@ export const visibleInVY = (el: Element) => {
 	return top >= 0 && bottom <= innerHeight;
 };
 
+export const isInView = (el: Element) => {
+	const { top, bottom, left, right } = el.getBoundingClientRect();
+	const { innerHeight, innerWidth } = window;
+	console.log(el, el.getBoundingClientRect(), innerHeight, innerWidth);
```

Comment:
```suggestion
	console.log(el, el.getBoundingClientRect(), innerHeight, innerWidth);
```

[whooppsi](https://en.wikipedia.org/wiki/1982_Harpoon_missile_misfire_incident)

Reply:
fixed